swift_browser_ui/server.py

```diff
@@ -157,6 +162,13 @@ async def servinit() -> aiohttp.web.Application:
         aiohttp.web.get('/sign/{valid}', handle_signature_request)
     ])
 
+    # Add token functionality
+    app.add_routes([
```

Comment:
Question: In order to use these endpoints, all it requires is for a login to be valid or also for additional permissions ?

Reply:
Yup, all that's required is a valid login – the tokens created are project scoped, so access to said project should be enough